GliaWidgets/Layout/NSLayoutConstraint.Extensions.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
```

Comment:
PureLayout is also available in this project. If the long term goal is to remove the dependency it is fine, but in Cortex we used no dependency-approach also: https://github.com/salemove/ios-cortex-demo/tree/master/Cortex%20Financial/Utility/Constrainable

Reply:
yes, we would like to have as less 3rd party as possible, but in sense of PureLayout I have a few ideas how to manage layout better, one of them is that PureLayout activates each constraint separately, which is not good in sense of performance. We have a few extensions on Core SDK side, but since we would like to avoid custom mechanisms/frameworks, so I decided to don't use anything additionally for now and use standard approach for managing constraints :) but it is discussable.